UnityProject/Assets/Scripts/Systems/Research/Objects/ResearchServer.cs

```diff
@@ -1,20 +1,18 @@
 ﻿using System;
 using System.Collections;
 using System.Collections.Generic;
-using System.IO;
-using Communications;
 using Items.Storage.VirtualStorage;
 using Mirror;
-using ScriptableObjects.Research;
 using UnityEngine;
 using Systems.Research.Data;
 using Shared.Systems.ObjectConnection;
+using Random = UnityEngine.Random;
 
 namespace Systems.Research.Objects
 {
 	public class ResearchServer : NetworkBehaviour, IMultitoolMasterable
 	{
-		[SerializeField] private int researchPointsTrickl = 25;
+		[SerializeField] private int researchPointsTrickle = 25;
```

Comment:
Does this not need a `FormerlySerializedAs` attribute?

Reply:
The value was never changed in inspector, so there's no need to keep the old reference.